packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -95,9 +93,11 @@ interface IProps extends React.BaseHTMLAttributes<HTMLDivElement> {
 	/**
 	 * Values used in displaying bottom-left icon
 	 */
-	stickerProps?: IClayStickerProps & {
-		content?: React.ReactNode;
-	};
+	stickerProps?:
+		| (IClayStickerProps & {
+				content?: React.ReactNode;
+		  })
+		| null;
```

Comment:
I think this is not an impediment but it is a little strange to have an API behavior that is different when it is set to `null`, I am probably not used to it... but maybe we can use a boolean here to solve this? it is more common in my view.

Reply:
I don't have any preference here, I did it in this way 'cause it felt weird to pass a boolean to a prop called `stickerProps`. It's like
- what time is it?
- yes

But you're more familiar with the rest of the code in Clay, so you decide @matuzalemsteles

Reply:
@carloslancha makes sense, that's a good argument 🙂! I don't have very strong opinions about it, the rest of our codebase doesn't have similar behaviors, so this would be the first behavior. I think we can make some changes about it, for example instead of just checking `null` we can set `stickerProps` to the default value with `{}` and just check in the conditional if there is any value for `stickerProps`.

Reply:
@matuzalemsteles I like that idea. I pushed some changes with that (that also fixes a wrong behavior with custom stickers not rendering default icon if no `children` or `content` is passed we didn't catch before. Please take a look and merge it if it's fine, I'm planing to cut a release later today. Thx!

Reply:
Obviously too late now, but I liked the first approach (using `null`) better.

- Boolean is no good for this because, like @carloslancha said, you don't want the answer to the question "what time is it?" to be "yes" or "no". A good heuristic for using a boolean is: are there clear semantics for what _both_ `true` and `false` should mean? In this example, `false` could communicate "no sticker", but `true` doesn't communicate a useful inverse.
- `null` is the value that JS uses to _explicitly_ signal "nothing here", as opposed to `undefined`, which just _implicitly_ means "I didn't put anything here" — so, for example, `undefined` can be useful to allow a default to take effect (pass nothing and you get a reasonable default), vs `null` to indicate "I want nothing here". In general, you also can't tell the difference between someone passing `undefined` and not passing anything — so `null` really is the right tool for _signalling_ "nothing".

Reply:
We didn't go with the `boolean` approach @wincent but what you say makes sense, we're passing `undefined` when we want nothing no be rendered but `null` is more explicit, thx!

@matuzalemsteles would you mind applying this feedback (use of `null` instead `undefined`) to https://github.com/liferay/clay/pull/3827 ? Thx!

Reply:
Makes sense! @carloslancha no problem, you want to get back with this behavior for the `ClayCardWithInfo`?

Reply:
@matuzalemsteles just add `null` as a valid value of `stickerProps`, that should be enough :)

Reply:
@matuzalemsteles did we added this change in the refactor pr?

Reply:
@carloslancha Sorry, I ended up forgetting to add this to the PR, I will send another one now, making this change. Thanks for remember.

Reply:
Thx @matuzalemsteles !

Reply:
Done!